Docs/QuickStartGuide.md

```diff
@@ -2,7 +2,7 @@
 
 This is a quick start guide to creating the most minimal Turbo iOS application from scratch get up and running in a few minutes. This will support basic back/forward navigation, but will not be a fully functional application.
 
-1. First, create a new iOS app from the Xcode File > New > Project menu and choose the default iOS "App" template.
+1. First, create a new iOS app from the Xcode File > New > Project menu and choose the default iOS "App" template. Note: When using XCode >= 12, be sure to choose "Storyboard" under "Lifecycle" and "UIKit App Delegate" under "Lifecycle" in the project creation dialog. 
```

Comment:
I think this meant to say:

Note: When using XCode >= 12, be sure to choose "Storyboard" under "Interface" and "UIKit App Delegate" under "Life Cycle" in the project creation dialog.

Reply:
damn, of course! thanks!